src/app/views/App.tsx

```diff
@@ -84,10 +85,27 @@ class App extends Component<IAppProps, IAppState> {
     });
   }
 
+  private collectDeviceCharacteristicsTelemetry() {
+    const deviceProperties = {
+      deviceHeight: screen.height,
```

Comment:
unfortunately, these dimensions are the device dimensions, not the dimensions that is available to the browser to use.
Here's the telemetry from my browser:
![image](https://user-images.githubusercontent.com/7122716/186457656-4fd5752d-9bf5-4ae9-bad7-f8f178b13251.png)

But using window.innerHeight and window.innerWidth I get 859 and 1553 because I have resized the window to better use my screen.

During a call we also discussed capturing window.devicePixelRatio to help identify where users are being bumped down to lower effective resolutions due to device settings.